spring-boot3-dummy/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.axonframework</groupId>
+        <artifactId>axon</artifactId>
+        <version>4.9.0-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>axon-spring-boot3-dummy</artifactId>
+    <name>Spring Boot 3 compiler dummy</name>
+    <description>
+        Module that contains shells of certain classes from Spring Boot 3 to allow dependent classes in the framework
+        to compile against those classes using JDK 8. This allows Axon Framework to maintain a baseline of JDK 8, while
+        providing specific Spring Boot 3 features that would otherwise require JDK 17 to compile.
+    </description>
+
+    <properties>
+        <!-- sonar analysis of these classes doesn't make sense -->
+        <sonar.skip>true</sonar.skip>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>testcontainers</artifactId>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <!-- exclude this jar from ever making into a public maven repository -->
```

Comment:
Curious: don't we want to do a similar exclusion for the JavaDoc?

Reply:
The javadoc would also be installed and deployed by the install and deploy plugins. Since they are de-activated, nothing should even make it to (central) repositories

Reply:
Awesome, so we're covered on that subject then.